chains/evm/calls/transactor/monitored/monitored.go

```diff
@@ -0,0 +1,206 @@
+package monitored
+
+import (
+	"context"
+	"math/big"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/rs/zerolog/log"
+
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls"
+	"github.com/ChainSafe/chainbridge-core/chains/evm/calls/transactor"
+)
+
+type RawTx struct {
+	nonce        uint64
+	to           *common.Address
+	value        *big.Int
+	gasLimit     uint64
+	gasPrice     []*big.Int
+	data         []byte
+	submitTime   time.Time
+	creationTime time.Time
+}
+
+type MonitoredTransactor struct {
+	txFabric       calls.TxFabric
+	gasPriceClient calls.GasPricer
+	client         calls.ClientDispatcher
+
+	maxGasPrice    *big.Int
+	increaseFactor *big.Int
```

Comment:
I think it is better to select another name for this variable since Factor is a divider that divides without a reminder. Also I would mention that this is basically a percentage